archive/Thematic/2021/Februrary/progressed/bridgecrew2/template.html

```diff
@@ -827,7 +863,13 @@ <h1 class="null" style="text-align: center;display: block;margin: 0;padding: 0;c
 															<td valign="top" class="mcnTextContent"
 																style="padding: 0px 18px 9px;line-height: 150%;mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;word-break: break-word;color: #111111;font-family: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;font-size: 18px;text-align: left;">
 
-																XXXX
+																<br>
```

Comment:
remove that first br. you can also open template.html  on your browser and check if the layout looks good for readers.